### PR TITLE
Tools: Fix errors in thermal calibration script

### PR DESCRIPTION
**Pull request by priseborough**
Fixes indentation error preventing baro calibration.
Fixes error in documentation

Calibration tested using pixracer log: https://logs.px4.io/plot_app?log=34161e52-ce1e-494c-8ce7-6f6a8e2f9182 gathered using SDLOG_MODE=2 and SDLOG_PROFILE=4

Parameter File:

[09_26_29.ulg.params.txt](https://github.com/PX4/Firmware/files/1611115/09_26_29.ulg.params.txt)

Plots:

[09_26_29.ulg.pdf](https://github.com/PX4/Firmware/files/1611118/09_26_29.ulg.pdf)

### CODE REVIEW

File: Tools/process_sensor_caldata.py [hunk 1]
```diff
@@ -94,11 +94,11 @@ def is_valid_directory(parser, arg):
         if sensor_instance == 0:
             sensor_baro_0 = d.data
             print('found baro 0 data')
-        num_baros = 1
+	    num_baros = 1
```
Comment (bkueng) [inline Tools/process_sensor_caldata.py]: The script uses a mix of spaces and tabs. Can you please convert everything to spaces (which is prefered for python)?

**Comment by dagar** [comment]
@bkueng thoughts on pep8 enforcement or similar?

**Comment by bkueng** [comment]
> @bkueng thoughts on pep8 enforcement or similar?

Sure, I use `pylint` for flight review: https://github.com/PX4/flight_review/blob/master/run_pylint.sh. The only downside is that from time to time it adds new checks (which is a good thing), but leads to unrelated CI failures.

**Comment by priseborough** [comment]
I've replaced the tabs, rebased and retested.

**Pull request merged**